app/api/api_v1/dependencies/auth.py

```
@@ -1,22 +1,159 @@
+                "message": "Inactive user",
+            },
         )
     return current_user
 
 
-async def get_current_admin(current_user: UserSchema = Depends(get_current_active_user)) -> UserSchema:
-    if current_user.role != 'admin':
+async def get_current_user_optional(
+    request: Request,
+    authorization: Optional[str] = Header(None),
+    db: AsyncSession = Depends(get_db),
```

Ruff (0.14.8)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)